README.md

```diff
@@ -139,6 +121,16 @@ Example command:
 
 Clone this repository to your desired folder:
 
+1.click: code and copy the link
+
+2.run: git clone
+
+3.Open the file in your code editor
+
+4.Right click on the html code nd click open live server
+
+5.Right click on the live server page and click inspect.
```

Comment:
- [ ] Also please update this section, since you're using webpack now, a tool like `Live server` wouldn't work. Instead, to open your application you need to run the development server with `npm start`. 🤓 🔧